test/tests/rails4.rb

```diff
@@ -555,7 +555,7 @@ def test_no_sql_injection_due_to_skipped_filter
   def test_command_injection_in_library
     assert_warning :type => :warning,
       :warning_code => 14,
-      :fingerprint => "9a11e7271784d69c667ad82481596096781a4873297d3f7523d290f51465f9d6",
+      :fingerprint => "21857e8872d187312a0b2470876bf6c8a8885df84c510d766f4639d95ae7cef7",
```

Comment:
Changed because the nested body parsing transforms the code string for the warning changes from:

```
s(:evstr, s(:lvar, :bad)))[[:class, :SweetLib]
```

to

```
s(:string_eval, s(:lvar, :bad)))[[:class, :SweetLib]
```

which affects the fingerprint.